### PR TITLE
Missing translation to fr

### PR DESCRIPTION
**Pull request by Gallyan**

### CODE REVIEW

**Review by andrey-helldar — APPROVED** [review]

**Review by andrey-helldar — APPROVED** [review]

File: locales/fr/fr.json [hunk 1]
```diff
@@ -272,7 +272,7 @@
     "If you already have an account, you may accept this invitation by clicking the button below:": "Si vous avez déjà un compte, vous pouvez accepter cette invitation en cliquant sur le bouton ci-dessous :",
     "If you did not create an account, no further action is required.": "Si vous n'avez pas créé de compte, vous pouvez ignorer ce message.",
     "If you did not expect to receive an invitation to this team, you may discard this email.": "Si vous n'attendiez pas d'invitation de cette équipe, vous pouvez supprimer cet e-mail.",
-    "If you did not receive the email": "If you did not receive the email",
+    "If you did not receive the email": "Si vous n'avez pas reçu cet email",
```
Comment (caouecs) [inline locales/fr/fr.json:275]: email => e-mail

pour la cohérence

Comment (caouecs) [inline locales/fr/fr.json:275]: email => e-mail

pour la cohérence

**Review by caouecs — CHANGES_REQUESTED** [review]

File: locales/fr/fr.json (hunk 1 above)
Comment (Gallyan) [reply locales/fr/fr.json]: C'est corrigé

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
Oh, sorry, I was restoring the branch name and forgot to move the PR before this. I'll fix it now.

**Review by WhereIsLucas — APPROVED** [review]

**Comment by andrey-helldar** [comment]
Merci

**Pull request merged**